public_html/wp-content/mu-plugins/blocks/source/components/post-list/inspector-controls.js

```diff
@@ -1,75 +1,61 @@
+/**
+ * External dependencies
+ */
+import { get }   from 'lodash';
+import PropTypes from 'prop-types';
+
 /**
  * WordPress dependencies
  */
-import { PanelBody, RangeControl } from '@wordpress/components';
-import { Component, Fragment }     from '@wordpress/element';
+import { RangeControl, PanelBody } from '@wordpress/components';
```

Comment:
Just curious why the order of listed components is changed here.

Reply:
That's a great question - either a merge conflict or I forgot how the alphabet works 😆

Reply:
😆